fiftyone/core/models.py

```
@@ -527,227 +518,100 @@ def _apply_image_model_data_loader(
+                        },
+                        label_field=label_field,
+                        confidence_thresh=confidence_thresh,
+                        classes=classes,
+                    )
+                    ctx.save(sample)
 
-            # Explicitly set in case actual # frames differed from expected #
-            pb.set_iteration(frame_counts[idx])
+                except Exception as e:
+                    if not skip_failures:
+                        raise e
```

Ruff (0.15.12)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)